### PR TITLE
Add Troubleshooting Section to .NET docs

### PR DESCRIPTION
**Pull request by MongoCaleb**
### Jira

https://jira.mongodb.org/browse/DOCSP-14591

### Staged Changes (Requires MongoDB Corp SSO)

https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/14591/sdk/dotnet/troubleshooting/

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/dotnet/troubleshooting.txt [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+Removing the following line from your .csproj file might help:
+
+.. code-block:: sh
+     :copyable: false
+
+     <Import Project="..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets" 
+      Condition="Exists('..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" />
+
+
+Troubleshooting WriteAsync Issues
```
Comment (nirinchev) [inline source/sdk/dotnet/troubleshooting.txt:85]: I'm not sure I understand this section - have you experienced this issue yourself?

Comment (MongoCaleb) [reply source/sdk/dotnet/troubleshooting.txt:85]: @nirinchev  - It came from the existing Realm docs troubleshooting section: https://docs.mongodb.com/realm-legacy/docs/dotnet/latest/index.html#troubleshooting

File: source/sdk/dotnet/troubleshooting.txt [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Getting a 'No Properties in Class' Exception
```
Comment (dacharyc) [inline source/sdk/dotnet/troubleshooting.txt:15]: Nit: We can probably write this without a gerund. Maybe something like `Resolve a "No Properties in Class" Exception`?

File: source/sdk/dotnet/troubleshooting.txt [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+---------------------------------------
+This common build failure is triggered when you have already built a 
+project and then add a new ``RealmObject`` sub-class. Choosing to 
+:guilabel:`Build` or :guilabel:`Run` your project does not rebuild the project 
+thoroughly enough to invoke the Fody Weaver. To fix this, 
+choose to :guilabel:`Rebuild` your project.
+
+.. _fail-to-weave:
+
+Failing to Weave
```
Comment (dacharyc) [inline source/sdk/dotnet/troubleshooting.txt:54]: Nit: we can probably rewrite this without a gerund, too. Maybe something like: `Diagnose Class Not Woven Issues`?

**Review by dacharyc — APPROVED** [review]
No show-stopping issues, but some of the copy here has passive voice, or long and complex sentences that are difficult to parse. I know this is porting over existing content, so I'm not sure how much you want to rewrite this to improve clarity and make it easier to read. If you feel like taking a pass at it, [Hemingway App](https://hemingwayapp.com/) is an easy tool to spot and fix writing clarity issues. Pasting this there highlights a lot of the issues. I didn't want to spend a lot of time pointing out small issues if you'd rather just merge and close this ticket, though.

**Pull request merged**